dev-docs/bidders/retailspot.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: bidder
+title: RetailSpot
+description: Prebid RetailSpot Bidder Adaptor
+pbjs: true
+pbs: false
+media_types: banner, video
+biddercode: retailspot
+gdpr_supported: true
```

Comment:
As Bret mentioned, if you claim `gdpr_support` your must provide a `gvl_id` here.

```
gvl_id: <your id here>
```